frontend/src/app/components/wp-inline-create/wp-inline-create.component.ts

```diff
@@ -58,6 +58,7 @@ import {WorkPackageChangeset} from "core-components/wp-edit/work-package-changes
 import {EditForm} from "core-app/modules/fields/edit/edit-form/edit-form";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 import {componentDestroyed} from "@w11k/ngx-componentdestroyed";
+import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3f0519806a21698d886f0aae816d7a75ed1548e8/frontend/src/app/components/wp-inline-create/wp-inline-create.component.ts#L62"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>